src/components/addCostumer/AddItemToCustomer.jsx

```diff
@@ -131,15 +146,13 @@ const AddItemToCustomer = () => {
           <h3 className="qty-sub-total">Sub total:&nbsp;{subTotal}</h3>
           <h1 className="qty-p-total">Grand total: &nbsp;{grandTotal}</h1>
         </div>
-        {/* <NavLink to="/customerpreview" style={{ textDecoration: 'none' }}> */}
         <button
           type="button"
           onClick={handleContribution}
           className="finish-btn"
         >
           Preview
         </button>
-        {/* </NavLink> */}
       </div>
```

Comment:
I will suggest that you remove comment code from your code base, this will make your code base cleaner.